lib/modules/apostrophe-global/index.js

```diff
@@ -182,6 +182,13 @@ module.exports = {
       label: 'Selecteer een polygon',
       choices: polygonsFormatted
     },
+    {
+      name: 'mapImageFlag',
+      type: 'select',
+      label: 'Welke kleur vlag moet er getoond worden op de plattegrond in het begroten overzicht en op de planpagina?',
+      choices: flagChoices,
```

Comment:
I don't see where the variable `flagChoices` is set

Reply:
It was already defined in the file, see line 26